src/main/java/com/sygic/travel/sdk/places/model/query/ToursGetYourGuideQuery.kt

```diff
@@ -0,0 +1,47 @@
+package com.sygic.travel.sdk.places.model.query
+
+/**
+ * ToursGetYourGuideQuery contains values which define the Get Your Guide tours to be fetched.
+ * To see what the parameters mean check the
+ * [API Documentation](http://docs.sygictravelapi.com/1.0/#section-tours).
+ */
+class ToursGetYourGuideQuery(
+	val query: String?,
+	val bounds: String?,
+	val parentPlaceId: String?,
+	val tags: String?,
+	val from: String?,
+	val to: String?,
+	val duration: String?,
+	val page: Int?,
+	val count: Int?,
+	val sortBy: SortBy?,
+	val sortDirection: SortDirection?
+) {
+
+	enum class SortBy constructor(sortBy: String) {
```

Comment:
put here `internal val` and you may ommit the rest. Also the property name could be something like `apiSortBy`

Reply:
I'm afraid I don't understand what you mean.

Reply:
`enum class SortBy constructor(internal val apiSortBy: String) {`

Reply:
By omitting the rest you mean omitting the `internal set` and `init`?